include/dem/explicit_euler_integrator.h

```diff
@@ -69,11 +69,20 @@ class ExplicitEulerIntegrator : public Integrator<dim>
    * to integrate
    * @param body_force A constant volumetric body force applied to all particles
    * @param time_step The value of the time step used for the integration
+   * @param momentum Momentum of particles
+   * @param force Force acting on particles
+   * @param MOI A container of moment of inertia of particles
+   * @param acceleration A container of acceleration of particles
    */
   virtual void
-  integrate_post_force(Particles::ParticleHandler<dim> &particle_handler,
-                       Tensor<1, dim>                   body_force,
-                       double                           time_step) override;
+  integrate_post_force(
+    Particles::ParticleHandler<dim> &        particle_handler,
+    Tensor<1, dim>                           body_force,
+    double                                   time_step,
+    std::unordered_map<int, Tensor<1, dim>> &momentum,
+    std::unordered_map<int, Tensor<1, dim>> &force,
+    std::unordered_map<int, double> &        MOI,
+    std::unordered_map<int, Tensor<1, dim>> &acceleration) override;
```

Comment:
I can understand why you store the force in a structure. But why acceleration? I mean, force = acceleration * mass. You can just store one of the two no?

Reply:
We need the previous step's acceleration in the Velocity Verlet integration. I changed it from a global container to a local container.